packages/evolution-frontend/src/components/pages/SurveyEndedPage.tsx

```
@@ -0,0 +1,63 @@
+                            {config.languages.map((language) => {
+                                return i18n.language !== language ? (
+                                    <a
+                                        href="#"
+                                        className="em"
+                                        key={`header__nav-language-${language}`}
+                                        onClick={(e) => {
+                                            e.preventDefault();
+                                            i18n.changeLanguage(language);
+                                            moment.locale(i18n.language);
+                                        }}
+                                    >
```

Biome (2.1.2)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)